tools/static-assets/scaffolds-js/index.js

```diff
@@ -0,0 +1,15 @@
+import { $$UpperName$$Collection } from './collection';
+import {
+  save$$UpperName$$,
+  update$$UpperName$$,
+  remove$$UpperName$$,
+  find$$UpperName$$ById
+} from './methods';
+
+export {
+  $$UpperName$$Collection,
+  save$$UpperName$$,
+  update$$UpperName$$,
+  remove$$UpperName$$,
+  find$$UpperName$$ById
+}
```

Comment:
Have you thought about using `export * from` instead? It's less of a burden to update it later.

Reply:
Agreed, and also it's cleaner